### ISSUE TITLE
Lossing buddy list groupings

### ISSUE THREAD

**Issue by advcomp2019** [jgeboski/purple-facebook#154]
I have all my Facebook friends in one group. Then at some random time, I find at least one maybe two of my Facebook friends moved back in the Buddies group. Then I move them back into my Facebook friends group. When I move them back, I do look at my offline Facebook users to see if there any more of the Facebook friends in the Buddies group, and it has no offline Facebook friends. Then later on, I find another random friend that I have to move back into the right group. I can't say how to reproduce this bug because it seems to be just random.

**Comment by dequis** [comment]
Would it be fine if this plugin just placed everyone in a "facebook" group?

**Comment by advcomp2019** [comment]
Not sure if that will help with this bug or not. I am still getting this bug at really random times. Plus I am not sure what is causing this yet. I have seen it with four of my friends most of the time right now.

**Comment by chuinker** [comment]
I'd be happy if the contacts defaulted to a "Facebook" group for now. There is too much online clutter for these users to be in "Buddies".

**Comment by brianjmurrell** [comment]
> Would it be fine if this plugin just placed everyone in a "facebook" group?

No.  I have my contacts grouped by type such as Friends, Family, Co-workers, etc.  Facebook might be only one of only several protocols I have for a single person and I take full advantage of Pidgin's contact merging capabilities to keep all of the protocols for a given person in that person's contact record.

Not being able to move Facebook contacts to a different groups **and have them stay there** completely defeat's Pidgin's contact merging.

**Comment by dequis** [comment]
Please try 0.9.5

https://github.com/dequis/purple-facebook/releases/tag/v0.9.5-9ff9acf9fa14
